api/tests/core/finance/test_commands.py

```diff
@@ -106,3 +110,36 @@ def test_move_siret(app):
 
     assert not src_venue.siret
     assert dst_venue.siret == siret
+
+
+@override_settings(SLACK_GENERATE_INVOICES_FINISHED_CHANNEL="channel")
+@clean_database
+def test_generate_invoices_internal_notification(app, css_font_http_request_mock):
+    venue = offerers_factories.VenueFactory()
+    batch = finance_factories.CashflowBatchFactory()
+    finance_factories.CashflowFactory.create_batch(
+        size=3,
+        batch=batch,
+        reimbursementPoint=venue,
+        status=finance_models.CashflowStatus.UNDER_REVIEW,
+    )
+    finance_factories.BankInformationFactory(venue=venue)
+
+    with patch("pcapi.core.finance.commands.send_internal_message") as mock_send_internal_message:
+        run_command(
+            app,
+            "generate_invoices",
+            "--batch-id",
+            str(batch.id),
+        )
+
+    assert mock_send_internal_message.call_count == 1
+    call_kwargs = mock_send_internal_message.call_args.kwargs
+    assert len(call_kwargs["blocks"]) == 1
+    call_block = call_kwargs["blocks"][0]
+    assert call_block["text"]["text"] == "La Génération de factures (VIR0) est terminée avec succès"
```

Comment:
Je vois que la CI ne passe pas, il faut éviter `VIR0` en dur mais construire le texte attendu dynamiquement, probablement (à tester) : 
```suggestion
    assert call_block["text"]["text"] == f"La Génération de factures ({batch.label}) est terminée avec succès"
```